notify-pull-request/action.yml

```diff
@@ -0,0 +1,367 @@
+name: "Notify PR"
```

Comment:
The duplication here is icky but it would be a lot of effort to re-write the actions logic in bash so this should do for now even if it's not optimal...